tools/debugging/stress_test_transfers.py

```diff
@@ -725,9 +725,6 @@ def main() -> None:
             ]
             raiden_args.extend(chain.from_iterable(node.items()))
 
-            if not is_checksum_address(address):
```

Comment:
Should this check if it's a valid hex address, so starts with '0x' and has 42 chars? But I guess for internal usage this is fine

Reply:
I changed it to `address = to_checksum_address(address)`. This will raise an exception if the address is invalid, and normalize it to be used with the REST APIs, without annoying the person using the test script.